src/coreclr/vm/debuginfostore.cpp

```diff
@@ -310,6 +357,61 @@ static void DoBounds(
     }
 }
 
+template <class T>
+static size_t DoBoundsCallback(
+    T trans, // transfer object.
+    ULONG32 cMap,
+    void* pContext,
+    size_t (*pfnWalkILOffsets)(ICorDebugInfo::OffsetMapping *pOffsetMapping, void *pContext)
+)
+{
+    CONTRACTL
+    {
+        THROWS;
+        GC_NOTRIGGER;
+        MODE_ANY;
+        SUPPORTS_DAC;
+    }
+    CONTRACTL_END;
+
+
+    // Bounds info contains (Native Offset, IL Offset, flags)
+    // - Sorted by native offset (so use a delta encoding for that).
+    // - IL offsets aren't sorted, but they should be close to each other (so a signed delta encoding)
```

Comment:
It didn't look like we were using signed delta encoding for IL offsets in the implementation.